### DISCUSSION TITLE
How do I fix the subtitles?

### DISCUSSION THREAD

**Discussion by ySergi** [crunchy-labs/crunchy-cli#401] [Q&A] [answered]
They got downloaded in a weird way, I don't know why they look like this:
![image](https://github.com/crunchy-labs/crunchy-cli/assets/138529583/525bacee-4778-4999-ab95-59d1160d7571)
How do I fix this?

PS: The command I used was: `.\crunchy-cli.exe download -a ja-JP -s es-ES -o "D:\Media\Crunchyroll\Lycoris Recoil\[S{season_number}E{episode_number}] {title}.mkv" https://www.crunchyroll.com/series/G1XHJV2X9/lycoris-recoil`

**Comment by bytedream** [comment]
On which episode does that happen? Which version of crunchy-cli are you using (use `crunchy-cli --version` to get it)? What is your video player?

**Reply by ySergi** [reply]
Practically in every episode it is happening.
The Crunchy-CLI version is the latest one. `.\crunchy-cli.exe --version
crunchy-cli 3.6.0`
The player is MPV, but that doesn't matter it occurs too in the Windows native player, but they're only in cursive and underscored:
![image](https://github.com/crunchy-labs/crunchy-cli/assets/138529583/3217c639-9835-4c83-bd75-bd25a4bfdb4f)

**Comment by bytedream** [comment, marked as answer]
I've identified the issue. It is caused by a library that manipulates the subtitles which was added in the latest version, I've created a patch and currently wait for it to be merged. When the fix lands in the library, I'll create a new crunchy-cli release.
Until then you can use [v3.5.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.2), the subtitles are downloaded correctly there.

**Reply by ySergi** [reply]
Alright, thanks!
